src/test/HelmetTest.js

```diff
@@ -221,16 +221,48 @@ describe("Helmet", () => {
                 expect(htmlTag.getAttribute("amp")).to.be.null;
             });
 
-            it("will not clear attributes handled outside of helmet", () => {
-                const htmlTag = document.getElementsByTagName("html")[0];
-                htmlTag.setAttribute("test", "test");
+            context("initialized outside of helmet", () => {
+                before(() => {
+                    const htmlTag = document.getElementsByTagName("html")[0];
+                    htmlTag.setAttribute("test", "test");
+                });
 
-                ReactDOM.render(
-                    <Helmet />,
-                    container
-                );
+                it("will not be cleared", () => {
+                    const htmlTag = document.getElementsByTagName("html")[0];
+
+                    ReactDOM.render(
+                        <Helmet />,
+                        container
+                    );
 
-                expect(htmlTag.getAttribute("test")).to.equal("test");
+                    expect(htmlTag.getAttribute("test")).to.equal("test");
```

Comment:
The other test cases render the Helmet, then set `htmlTag`, I feel like it doesn't matter.